bundles/org.openhab.binding.mqtt.awtrixlight/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,593 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mqtt"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="awtrixclock">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="broker"/>
+		</supported-bridge-type-refs>
+		<label>Awtrix Light Device</label>
+		<description>Device with Awtrix Light firmware</description>
+		<category>screen</category>
+		<channels>
+			<channel id="app" typeId="app"/>
+			<channel id="autoBrightness" typeId="autoBrightness"/>
```

Comment:
Both channel id and type id should be `lower-case-hyphen`

Reply:
Done